### PR TITLE
RosterPage: Clear TextFields after closing AddContactSheet

### PR DESCRIPTION
**Pull request by JBBgameich**

### CODE REVIEW

File: src/qml/RosterAddContactSheet.qml [hunk 1]
```diff
@@ -54,7 +59,10 @@ Kirigami.OverlaySheet {
 
 			Controls.Button {
 				text: qsTr("Cancel")
-				onClicked: close()
+				onClicked: {
+					clearInput();
+					close()
```
Comment (LNJ2) [inline src/qml/RosterAddContactSheet.qml]: Missing `;`

File: src/qml/RosterAddContactSheet.qml [hunk 2]
```diff
@@ -63,7 +71,8 @@ Kirigami.OverlaySheet {
 				enabled: jidField.length > 0
 				onClicked: {
 					kaidan.rosterController.addContact(jidField.text, nickField.text);
-					close();
+					clearInput();
+					close()
```
Comment (LNJ2) [inline src/qml/RosterAddContactSheet.qml]: Missing `;`

**Comment by lftkv** [comment]
@JBBgameich GitHub direkt upload

**Comment by LNJ2** [comment]
@lftkv He could have used it, but from the command line it's also no big problem. Everyone can do it as he/she wants. :)

**Comment by TheWildBear** [comment]
@lftkv Yep, der ist mit github verbunden... 🤔 aber wie kommt der darauf? 🤔🤔🤔

**Pull request merged**